packages/expo/package.json

```diff
@@ -52,16 +53,16 @@
     "@types/react": "*",
     "@types/react-dom": "*",
     "eslint-config-custom": "*",
-    "expo-auth-session": "^5.4.0",
```

Comment:
❓  Is it safe to remove this package from the devDependencies? 🤔

Reply:
you are correct. It should not be removed. i missed it during the last rebase. I am trying to resolve another issue. so please wait until i ping you to avoid wasting time reviewing this. :(